src/scrape_up/github/respository.py

```diff
@@ -205,4 +205,16 @@ def get_pull_requests_ids(self):
             return pull_requests_ids           
         except:
             message = "No pull requests found"
-            return message
\ No newline at end of file
+            return message
+        
+    def commits(self):
```

Comment:
Also, include this method in the `documentation. md` file.

Reply:
done ✅